### DISCUSSION TITLE
How Can I edit issues or add comments to issues using common markdown format

### DISCUSSION THREAD

**Discussion by Pengfei73** [ankitpokhrel/jira-cli#751] [Q&A]
When I try to update issues using editor,
Input texts to it,
for example:
```
### My title
```
then submit,
The final result shown on web UI is
```
1. 1. 1. My title
```
which is translated by jira-markdown.

Is it possible to use the common markdown format or github-flavored markdown instead of jira-markdown?
How to config my jira-cli, I fail to find neither related docs, instructions, nor discussions.

**Comment by ankitpokhrel** [comment]
The tool already support github-flavored markdown. 

Could you please provide more information like if you are on cloud or using on-premises installation? and what's the version of Jira you're using?
